php/7.2/fpm-alpine-develop-xdebug/conf/php.ini

```diff
@@ -1,68 +0,0 @@
-;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
-;; Stock PHP Settings
-;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
-
-; This sets the maximum amount of memory in bytes that a script is allowed to allocate. 
-; This helps prevent poorly written scripts for eating up all available memory on a server. 
-; Note that to have no memory limit, set this directive to -1.
-memory_limit = 4G
-
-; This sets the maximum time in seconds a script is allowed to run before it is terminated by the parser. 
-; This helps prevent poorly written scripts from tying up the server. 
-; The default setting is 30. When running PHP from the command line the default setting is 0.
-max_execution_time = 30
-
-date.timezone = UTC
-
-; The maximum size of an uploaded file. When an integer is used, the value is measured in bytes. 
-; Shorthand notation may also be used.
-upload_max_filesize = 20M
-
-; Sets max size of post data allowed. This setting also affects file upload. To upload large files, 
-; this value must be larger than upload_max_filesize. Generally speaking, memory_limit should be 
-; larger than post_max_size. When an integer is used, the value is measured in bytes. Shorthand notation may also be used.
-post_max_size = 20M
-
-; Whether to transparently compress pages. If this option is set to "On" in php.ini or the Apache configuration, 
-; pages are compressed if the browser sends an "Accept-Encoding: gzip" or "deflate" header. 
-; "Content-Encoding: gzip" (respectively "deflate") and "Vary: Accept-Encoding" headers are added to the output. 
-zlib.output_compression = On
-
-display_errors = stderr
-error_log = /dev/stderr
-display_startup_errors = On
-log_errors = On
-
-; Where the sendmail program can be found, usually /usr/sbin/sendmail or /usr/lib/sendmail. 
-; configure does an honest attempt of locating this one for you and set a default, 
-; but if it fails, you can set it here.
-sendmail_path = ${PHP_SENDMAIL_PATH}
-
-;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
```

Comment:
7.2 uses an old version of xdebug, we need these older settings to remain the same.